packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -149,7 +158,10 @@ export const Headline = styled.h3`
 
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     ${({ script, topStory }) =>
-      script && (topStory ? getParagon(script) : getGreatPrimer(script))}
+      script &&
```

Comment:
With this https://github.com/bbc/psammead/pull/1987#discussion_r321116701 suggestion you could simplify this to

```js
${({ script, topStory }) => script && getHeadlineFontStyle(script, topStory)}
```